README.md

```diff
@@ -5,16 +5,31 @@ We hope the process is going well for you so far.
 
 Please follow the instructions below, and don't hesitate to ask us for anything you need.
 
+The dev task is currently available in the following programming languages:
+* Python (both `the_big_bang.py` and `the_big_bang.ipynb` are available)
+* Go (`big_bang.go` and `big_bang_test.go`)
+* Node.js (`big_bang.js`)
+
+
 ## Instructions
 
+If you prefer to develop locally: 
+1. Clone the repo
+2. Choose the relevant file based on your preferred programming language
+3. Follow the instructions in the file
+
+
+If **Python** is your favorite language, and you prefer working with Jupyter Notebook, you can also use **Binder**:
```

Comment:
Do we want this to appear *before* the local option? personally I think it's more fun to do it in jupyter...

Reply:
IMO it's better to go from general to specific (also - lately most of the candidates preferred to work locally).